packages/framework/src/Component/Redis/RedisFacade.php

```diff
@@ -11,19 +11,45 @@ class RedisFacade
     /**
      * @var \Redis[]
      */
+    protected $allClients;
+
+    /**
+     * @var \Redis[]
+     */
+    protected $persistentClients;
+
+    /**
+     * @deprecated This property is deprecated since SSFW 7.3
+     * @var \Redis[]
+     */
     protected $cacheClients;
 
     /**
-     * @param \Redis[] $cacheClients
+     * @param \Redis[] $allClients
+     * @param \Redis[] $persistentClients
      */
-    public function __construct(array $cacheClients)
+    public function __construct(iterable $allClients, iterable $persistentClients = [])
```

Comment:
shouldnt we keed there cacheClients variable name?

Reply:
Why? The meaning of the parameter has changed, because now `$cacheClients = $allClients - $persistentClients`. This change of meaning is still backward compatible because when you don't provide `$persistentClients` it holds true that `$cacheClients = $allClients`.

There might be a problem with the rename if they are using the named argument configuration, but I can't think of any better way how to handle this...